src/main/java/io/pinecone/clients/Index.java

```diff
@@ -779,6 +780,207 @@ public DescribeIndexStatsResponse describeIndexStats(Struct filter) {
         return blockingStub.describeIndexStats(describeIndexStatsRequest);
     }
 
+    /**
+     * {@inheritDoc}
+     * <p> Example</p>
+     *  <pre>{@code
+     *      import io.pinecone.clients.Index;
+     *      import io.pinecone.clients.Pinecone;
+     *      import io.pinecone.proto.ListResponse;
+     *
+     *      Pinecone pc = new Pinecone.Builder(System.getenv("PINECONE_API_KEY")).build();
+     *      String indexName = "example-index";
+     *      Index index = pc.getIndexConnection(indexName);
+     *      ListResponse listResponse = index.list("example-namespace");
+     *      System.out.println(listResponse);
```

Comment:
nit: I would lose the `System.out.println()` statements in all of these. I don't think there's much value in it and we're not doing it elsewhere as far as I know.